iroh/Cargo.toml

```diff
@@ -15,7 +15,7 @@ rust-version = "1.65"
 [dependencies]
 anyhow = { version = "1", features = ["backtrace"] }
 bao-tree = { version = "0.5.0-rc.1", features = ["tokio_fsm"], default-features = false }
-derive_more = { version = "0.99.17", git = "https://github.com/JelteF/derive_more", features = ["debug", "display", "from", "try_into"] }
+derive_more = { package = "derive_more_preview", version = "0.1.0", features = ["debug", "display", "from", "try_into"] }
```

Comment:
Sigh. I guess we can yank it once they release it?

Reply:
some day, in many moons